service/src/main/java/uk/nhs/adaptors/gp2gp/common/service/ConfidentialityService.java

```diff
@@ -15,12 +15,9 @@
 public class ConfidentialityService {
     private static final String REDACTION_CODE_SYSTEM = "http://hl7.org/fhir/v3/ActCode";
     public static final String NOPAT = "NOPAT";
-    private static final String REDACTION_CONFIDENTIALITY_CODE = """
-        <confidentialityCode
-            code="NOPAT"
-            codeSystem="2.16.840.1.113883.4.642.3.47"
-            displayName="no disclosure to patient, family or caregivers without attending provider's authorization"
-        />""";
+    public static final String REDACTION_CONFIDENTIALITY_CODE = "<confidentialityCode code=\"NOPAT\" "
```

Comment:
This static constant is already in [ConfidentialityCodeUtility.java](https://github.com/NHSDigital/integration-adaptor-gp2gp-sending/pull/1083/files#diff-004db1f857aaf7f196223df7eccf0871d1c47221d8ff269cda7d006fff5246ee)?  Can we reuse that instead?